HPS/Heartland/view/adminhtml/web/js/view/payment/securesubmit.js

```diff
@@ -0,0 +1,281 @@
+
+
+/*
+ *  Heartland payment method model
+ *
+ *  @category    HPS
+ *  @package     HPS_Heartland
+ *  @author      Charlie Simmons <charles.simmons@e-hps.com>
+ *  @copyright   Heartland (http://heartland.us)
+ *  @license     https://github.com/hps/heartland-magento2-extension/blob/master/LICENSE.md
+ */
+
```

Comment:
Below, `document.querySelector` won't always return a node, so the return value should be checked before accessing/mutating properties.